ansible/roles/test/files/ptftests/hash_test.py

```diff
@@ -85,11 +85,15 @@ def check_hash(self, hash_key):
 
         hit_count_map = {}
         for _ in range(0, self.BALANCING_TEST_TIMES):
+            in_port = random.choice([port for port in self.src_ports if port not in exp_port_list]) if hash_key == "ingress-port" else in_port
```

Comment:
in loose mode, you only need to iterate once for every port, and then check if you receives packet on multiple ports. you do not need to send 10000 packets. 

in fact, such test can be used for all hash field, and in this case, you just verify if the asic actually used the hash field as a load-balancing factor.

Reply:
done